src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -309,7 +309,10 @@ public static String bannerErrorMessage() {
    * Click on the Plugin group title: Sink
    */
   public static void clickSink() {
-    ElementHelper.clickOnElement(CdfStudioLocators.sink);
+    // Check if Sink is not already expanded
+    if (!ElementHelper.isElementDisplayed(CdfStudioLocators.bigQueryObject)) {
```

Comment:
I think it would be better to add another method like "expandSinkPluginGroup" and then add this check. This method is only meant for clicking on the sink group and not ensuring whether it is expanded.

Reply:
Sounds good, in that case all the `When sink is ...` steps should be updated to use that method.